src/core/namespace.h

```diff
@@ -130,6 +130,7 @@ typedef struct NamespaceParameters {
 
         const char *root_directory;
         const char *root_image;
+        int root_directory_fd;
```

Comment:
If you move it above root_directory it saves one padding and express the priority better ;)